.travis.yml

```diff
@@ -31,30 +51,44 @@ addons:
       - openssh-client
       - openssh-server
       - libopenmpi-dev
+      - clang
+      - clang-tidy-5.0
+      - cmake-data
+      - graphviz
+  homebrew:
+    packages:
+      - gcc@7
+      - graphviz
```

Comment:
why?

Reply:
same as above, graphviz seems redudant at this point. It's within my plan to output allreduce toplogy graph as some point in next couple of prs.